frontend-react/src/pages/Hero.jsx

```diff
@@ -1,7 +1,75 @@
 import React from 'react';
+import { useTypewriter, Cursor } from 'react-simple-typewriter';
+import { motion } from 'framer-motion';
 
-const Hero = () => (
-  <div>Hero</div>
-);
+import image from '../assets/index.js';
+import { Rings } from '../components';
+
+const Hero = () => {
+  const [text] = useTypewriter({
+    words: ["Hi, I'm a Web Developer", 'and a Gamer', ' and a Programmer'],
+    loop: 0,
+    delaySpeed: 2000,
+    deleteSpeed: 75,
+    typeSpeed: 75,
+  });
+
+  const scaleVariants = {
+    whileInView: {
+      scale: [0, 1],
+      opacity: [0, 1],
+      transition: {
+        duration: 1,
+        staggerChildren: 1,
+        times: [0, 0.1, 1],
+        ease: 'easeInOut',
+      },
+    },
+  };
+// mt-[210px] mb-[210px]"
+  return (
+    <div
+      id="hero"
+      className="relative text-center flex flex-col items-center justify-center mt-[210px] mb-[210px]"
+    >
+      <Rings />
+      <img
+        src={image.profilePic}
+        alt="lightLogo"
+        className="w-[300px] h-[300px] rounded-full  object-cover"
+      />
+      <div className="overflow-hidden">
+        <h1 className="mt-4 mb-4 font-bold text-3xl text-[#F7AB0A] tracking-[15px]">
+          Daniel Ye
+        </h1>
+        <h2>
+          <span className="text-white">{`< ${text} />`}</span>
+          <Cursor cursorColor="#F7AB0A" />
+        </h2>
+      </div>
+      <motion.div
+        variants={scaleVariants}
+        whileInView={scaleVariants.whileInView}

```

Comment:
For whileInView, you only have to mention the name as you have already told the framer motion that you'll be using a variant called scaleVariants:
```javascript
<motion.div
   variants={scaleVariants}
   whileInView="whileInView"
>
```
It would be great if you change the name. **whileInview="whileInView"** doesn't make a lot of sense. It can confuse. Instead of that, something like **whileInView="show"** sounds good.